DigitalSlideArchive/ALBench
# Organize pytest

A few changes to the pytest tests:

1. Separate the five `test_*` functions, each into its own `test_*.py` file.
2. Place supporting functionality in new files, `check.py`, `create.py`, and `exercise.py`.
3. Rename the tests with numbers so that they will be executed in a reasonable order.
4. Make the test for needed external packages explicitly test for each needed component of each package.

Addresses Issue #16.
